test/test_init_diagnostics.rb

```diff
@@ -139,11 +139,17 @@ def test_data_adapter_init
     markers = metadata['markers']
     assert_marker_equal(markers[0], "overall", "start")
     assert_marker_equal(markers[1], "data_store", "start", "load")
-    assert_marker_equal(markers[2], "data_store", "end", "load", true)
-    assert_marker_equal(markers[3], "get_id_lists", "start", "network_request")
-    assert_marker_equal(markers[4], "get_id_lists", "end", "network_request", 200)
-    assert_marker_equal(markers[5], "overall", "end")
-    assert_equal(6, markers.length)
+    assert_marker_equal(markers[2], "download_config_specs", "start", "fetch_from_adapter")
```

Comment:
wait this changes the schema of diagnostics?

Reply:
I added steps for the adapter flow. The schema should be the same (key, action, step, value)